### ISSUE TITLE
Mixed keras imports causing version conflicts

### ISSUE THREAD

**Issue by iAmGiG** [iAmGiG/BotnetTrafficAnalysisFederaedLearning#15]
## Problem

Code mixes standalone Keras imports with TensorFlow Keras imports, which can cause version conflicts and unpredictable behavior.

**Example in train_og.py:**
```python
from keras.models import load_model          # Standalone keras
from tensorflow.keras.models import Model     # TensorFlow keras
from tensorflow.keras.layers import Input, Dense
```

**Files affected:**
- `anomaly-detection/train_og.py`
- `anomaly-detection/test.py`
- Other training scripts

## Fix

Use `tensorflow.keras` exclusively:
```python
from tensorflow.keras.models import load_model, Model
from tensorflow.keras.layers import Input, Dense
```

## Priority

MODERATE - Can cause model loading failures and compatibility issues

**Comment by iAmGiG** [comment]
✅ FIXED and TESTED

**What was done:**
- Standardized all keras imports to use `tensorflow.keras`
- Files fixed:
  - anomaly-detection/train_autoencoder.py:9
  - anomaly-detection/test_autoencoder.py:6
  - classification/train_classifier.py:13-16
  - classification/test_classifier.py:10

**Changes:**
- Changed `from keras.models import load_model` → `from tensorflow.keras.models import load_model`
- Changed `from keras.layers import Dense` → `from tensorflow.keras.layers import Dense`
- Consolidated all imports to use tensorflow.keras namespace

**Testing:**
- All scripts import successfully
- Classification training: 99.85% accuracy
- Anomaly detection training: completed successfully
- No version conflicts observed

**Status:** Ready for TensorFlow 2.x environments
